frontend/src/pages/ForgotPasswordPage.tsx

```diff
@@ -0,0 +1,152 @@
+import { useState, FormEvent } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { ModalContent } from '../components/modals/modalparts/ModalContent';
+import { ModalHeader } from '../components/modals/modalparts/ModalHeader';
+import { Footer } from '../components/Footer';
+import { paths } from '../routers/paths';
+import { Input } from '../components/forms/FormField';
+import { ReactComponent as MailIcon } from '../icons/mail_icon.svg';
+import { api } from '../api/api';
+import css from './ForgotPassword.module.scss';
+
+const classes = classNames.bind(css);
+
+export const ForgotPasswordPage = () => {
+  const { t } = useTranslation();
+  const defaultDisableTime = 30; // Seconds
+
+  const [view, setView] = useState('Send link');
+  const [email, setEmail] = useState('');
+  const [resendDisabled, setResendDisabled] = useState(false);
+  const [disableTime, setDisableTime] = useState(defaultDisableTime);
+
+  const sendEmailLink = () => {
+    api.sendPasswordResetLink(email);
+  };
```

Comment:
The possible errors should be checked. For example unknown email causes 404, but the page shows as if all went fine.